### PR TITLE
Added Learn page with some starting content

### PR DESCRIPTION
**Pull request by mtharrison**
Please see issue #166 for some more context.

I've added a Learn page with some subsections, styling/navigation is based on the plugins page. In issue #119 @nlf said he'd consider folding tutorials into this section too. Not sure what the thoughts are on that now. Personally I think it's nice to have the tutorials a single click away from the homepage but can see the benefit of combining them too.

Please feel free to throw suggestions or more content for this section at me and I'll add some more commits.

### CODE REVIEW

**Comment by AdriVanHoudt** [comment]
I'm not a fan of the title `learn` since that hints more to tutorials, I would rather go with `external resources` or something 
Also might want to add https://gist.github.com/hueniverse/a06f6315ea736ed1b46d to the gists

**Comment by mtharrison** [comment]
An issue with that name is that it wouldn't fit well into the top navigation. I think it would be a shame to demote the new section to the footer nav only.

So the options I can see are:

1. Go with **External Resources** and demote to footer nav only
2. Go with **Learn** and merge in tutorials
3. Add the content from this new section under another title, **Need Help?** seems to make sense because it already links to Postmile as an example application and Make Me Hapi.

**Comment by AdriVanHoudt** [comment]
Hmm how about just finding a new name? Like `Books` or `Resources`?

**Comment by mtharrison** [comment]
A whole section for **Books** wouldn't work at the moment, with only 1 book. **Resources** might be better, but I still think **Learn** is more descriptive of the content being featured. **Resources** sounds like it could be anything really. But it's your call ultimately, I appreciate you letting me add my book in any way :)

**Comment by AdriVanHoudt** [comment]
Good points. And this is in no way my call I'm just giving my opinion.

**Comment by nlf** [comment]
maybe `Resources` is the better name for this, then it becomes easier to differentiate between internal and external tutorials too.. i think if you rename learn to resources, this is ready to go. thanks!

File: templates/resources.jade [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                p= item.description
+
+    .column.nav-secondary
+      nav(role='toc')
+        h3 Resources
+
+        ul.unstyled(role='categories')
+          each category, categoryName in resources.categories
+            li(role='category')
+              a(href='##{categoryName}')= categoryName
```
Comment (AdriVanHoudt) [inline templates/resources.jade]: newline here (if I can be picky ^^)

Comment (mtharrison) [inline templates/resources.jade]: Sure

**Comment by nlf** [comment]
thanks!

**Pull request merged**